src/Label/Events/Excluded.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Label\Events;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+
+final class Excluded extends AbstractEvent
+{
+    public static function deserialize(array $data): Excluded
+    {
+        return new self(
+            new UUID($data[self::UUID]),
+            $data[self::NAME]
```

Comment:
I don't think we don't need to store the name of the label. This is a bit of the old approach that tries to reuse a common base class. I would just create a very simple class with no inheritance that only stores the UUID of the label.